app/models/content_module.rb

```diff
@@ -9,13 +9,14 @@ class ContentModule
 
   attr_accessor :shortcode, :version, :version_description, :title, :description_md,
                 :short_description, :released_at, :materials_url, :professional, :difficulty,
-                :platform, :language, :editor, :domains, :categories, :who_is_this_for_md,
+                :platform, :language, :editor, :domains, :categories, :who_is_this_for_md, :module_outcomes_md,
                 :covered_concepts_md, :authors, :lessons, :git_commit_hash, :card_artwork_image,
                 :featured_banner_image, :twitter_card_image, :root_path, :access_personal,
                 :access_team
 
   attr_markdown :who_is_this_for, source: :who_is_this_for_md, file: false
   attr_markdown :covered_concepts, source: :covered_concepts_md, file: false
+  attr_markdown :outcomes, source: :module_outcomes_md, file: false
   attr_markdown :description, source: :description_md, file: false
   attr_image :card_artwork_image_url, source: :card_artwork_image, variants: %i[original w560 w240]
```

Comment:
Nothing to do with the actual review, but I think we talked about not requiring `card_artwork_image` anymore, as it is identical to `feature_banner` after rebranding?

Reply:
@jellodiil Can you please also suggest what artwork would be there for content modules? or lesson or segment?

Reply:
think it is just `feature_banner` and `twitter_card` . 
Feature_banner will be used everywhere where `card_artwork` used to be (they used to be two slightly different images, but now its the same image for both, but I think we exported those two at different sizes).